extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -403,19 +411,31 @@ export async function makeHttpRequest(account: AzureAccount, subscription: azure
 		requestUrl = `${account.properties.providerSettings.settings.armResource.endpoint}${path}`;
 	}
 
-	let response: AxiosResponse | undefined;
+	let response;
 	switch (requestType) {
 		case HttpRequestMethod.GET:
-			response = await axios.get(requestUrl, config);
+			response = await httpClient.sendGetRequestAsync<HttpClientResponse>(requestUrl, {
+				headers: {
```

Comment:
Can't you just use networkRequestOptions for this and DELETE as well?

Reply:
networkRequestOptions contains a body field, though which is not expected in GET and DELETE requests